DOCS/interface-changes.rst

```diff
@@ -80,6 +80,7 @@ Interface changes
       certain unreliable video metadata. Also flip the defaults of all builtin
       HW deinterlace filters to always deinterlace.
     - change vf_vavpp default to use the best deinterlace algorithm by default
+    - --ab-loop-* options now use relative time, like --start.
```

Comment:
documentation should be contained in corresponding commit to ensure consistency

Reply:
squashed. the old commits removing opt_time are still available at https://github.com/martinetd/mpv/commits/remove_opt_time but will need quite a bit more work and complexify code quite a bit, I'm not sure it's worth it.